### ISSUE TITLE
mimic-cxr with mimic-iii

### ISSUE THREAD

**Issue by OPMEDICS** [MIT-LCP/mimic-code#730]
There is a one-year overlap between mimic-cxr and mimic-iii, is there any subjects in both datasets that we can match the cxr to the clinic data?

**Comment by alistairewj** [comment]
No, this is not possible. The identifiers are intentionally made distinct. We plan to publish an updated clinical dataset in the future.

**Comment by Seripad** [comment]
Hi Alistair,
I have some questions and would appreciate your help. In the video tutorial, you mentioned MIMIC IV includes a high number of neonate patients. 
1-I found that  60879  numbers of patients have an anchor-age of zero. I think they are neonates right? but in  MIMIC  IV CXR there were not any subject ID with an anchor-age of zero. I am wondering if the CXR file of the database includes any neonates patients?
if no what kind of information associated with neonates cases are available in the dataset?
2- I also went through the  MIMIC III articles. Does MIMIC include CXR images of the patients?
Thank you
